### PR TITLE
che #9062: Adding info about env var for setting workspace pod termination grace period for Kubernetes / OpenShift infrastructures

### PR DESCRIPTION
**Pull request by ibuziuk**
Doc PR for https://github.com/eclipse/che/pull/9118

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: src/main/pages/setup-kubernetes/kubernetes-config.md [hunk 1]
```diff
@@ -66,4 +66,11 @@ When Eclipse Che gets deployed to Kubernetes, a PVC `che-data-volume` is [create
 * `kubectl describe pvc che-data-claim`, find PV it is bound to, then `oc describe pv $pvName`, you will get a local path with logs directory. Be careful with permissions for that directory, since once changed, Che server wont be able to write to a file
 * in Kubernetes web console, eclipse-che namespace, **pods > che-pod > logs**.
 
+## Che Workspace Termination Grace Period
+
+Grace termination period of Kubernetes / OpenShift workspace's pods defaults '0', which allows to terminate
```
Comment (sleshchenko) [inline src/main/pages/setup-kubernetes/kubernetes-config.md:71]: Please add a note that user-defined termination period will be used if it is specified.

**Review by sleshchenko — APPROVED** [review]
Other LGTM

File: src/main/pages/setup-kubernetes/kubernetes-config.md (hunk 1 above)
Comment (ibuziuk) [reply src/main/pages/setup-kubernetes/kubernetes-config.md:71]: good point. lemme update

Comment (sleshchenko) [reply src/main/pages/setup-kubernetes/kubernetes-config.md:71]: Also please add the same information to [che.properties](https://github.com/eclipse/che/pull/9118/files#diff-2de0b0a873741f8e0d0879a46f8d5266R419) file.

Comment (ibuziuk) [reply src/main/pages/setup-kubernetes/kubernetes-config.md:71]: sure, that's the plan ;-)

**Comment by ibuziuk** [comment]
@sleshchenko have added info about ` user-defined termination period` in both doc and property file

**Pull request merged**